src/config/ServerlessLayersConfig.js

```diff
@@ -1,6 +1,21 @@
 class ServerlessLayersConfig {
-  constructor(options) {
+  constructor(options, slsVersion) {
+    console.log(`[ LayersPlugin ]: - options are: ${JSON.stringify(options)}, sls version is ${slsVersion}`);
+
+    if (slsVersion.startsWith('3') && Object.entries(options).length > 0) {
+      const v3Options = {};
+      options.param.forEach((v3Option) => {
```

Comment:
what is `options.param` ? can we guarantee it's always there? you check for options entries length, but not for the existing of `param` key

Reply:
When using sls v3, the options are not written as --key value but with --param="key=value". Added a little check for this.